README.md

```diff
@@ -5,6 +5,16 @@ author: dvana
 ---
 #Get started with Azure Data Catalog
 
-This sample shows you how to Register, Search, and Delete a data asset using the Data Catalog REST API.
+This sample shows you how to use the Azure Data Catalog REST API to Register, Search, and Delete data assets.
+
+To get started using this sample, you first need to build the executable, following below steps:
+
+* Download the sample project
+* Open it in Visual Studio and install the latest version of dependent NuGet packages (e.g Microsoft.IdentityModel.Clients.ActiveDirectory) 
+* Create (or use and existing) Azure Active Directory Application ID and the corresponding information. If you do not have one and need instructions on how to get one, you can check [here](https://docs.microsoft.com/en-us/rest/api/datacatalog/register-a-client-app) 
+* Compile the app.
+* You are now ready to run it.
```

Comment:
Resolved.

Reply:
The app file name in the sample is very descriptive, so that would be a very straightforward thing to be spotted by the developer audience. Hence, resolving as By Design.

Reply:
Please help me understand how the dev is harmed by a sentence of the form "To use the sample please open the Program.cs file and examine the contents to see how the REST APIs are used."?